app/mailers/audit_objects_mailer.rb

```diff
@@ -0,0 +1,15 @@
+# Send report of invalid objects to admin team
+class AuditObjectsMailer < ActionMailer::Base
```

Comment:
the other two must be using a nil layout (if they arent providing one?)? Wonder if it would hurt just to make them use the mailer layout?

All good though. Approving now.

Also fun fact, you can setup mailer previews if you want since rails 4.1: http://edgeguides.rubyonrails.org/action_mailer_basics.html#previewing-emails

But thats out of scope haha